.prettierignore

```diff
@@ -1,6 +1,4 @@
-dist/
-.cache/
-**/coverage/
-**/node_modules/**
-/node_modules
-/package-lock.json
\ No newline at end of file
+dist
+.cache
+coverage
+node_modules
```

Comment:
If we just let the commit hook do its thing then we are only running on `.js` and `.ts` files, as that's what's defined in the `lint-staged` section on the root `package.json`
```
"**/*.{js,ts}": [
  "eslint --fix",
  "prettier --write"
],
```
I'm imagining when we get to the point that we'd be happy to run prettier on the entire codebase we'd add the following extra `package.json` scripts:
```
"prettier": "prettier --write \"./**/*.{ts,js}\"",
"prettier:check": "prettier --check \"./**/*.{ts,js}\"",
```
and we could add `prettier:check` to the existing `lint:all` script.

Maybe safer to just leave `package-lock.json` just in case though. I'll re-add.

Reply:
Ah I see, I didn't think of that. I'm happy then, thanks @toddym42